tests/ops/test_stats.py

```diff
@@ -114,6 +114,27 @@ def test_autocorrelation():
     )
 
 
+def test_autocorrelation_trivial():
+    x = torch.zeros(10)
+    with xfail_if_not_implemented():
```

Comment:
i guess this xfail is no longer required?

Reply:
sure, let me try removing it :+1: